src/source/Rtcp/RtcpPacket.c

```diff
@@ -14,7 +15,7 @@ STATUS setRtcpPacketFromBytes(PBYTE pRawPacket, UINT32 pRawPacketsLen, PRtcpPack
     // We don't assert exact length since this may be a compound packet, it
     // is the callers responsibility to parse subsequent entries
     packetLen = getInt16(*(PUINT16) (pRawPacket + RTCP_PACKET_LEN_OFFSET));
-    CHK((packetLen + 1) * RTCP_PACKET_LEN_WORD_SIZE >= pRawPacketsLen, STATUS_RTCP_INPUT_PACKET_LEN_MISMATCH);
+    CHK((packetLen + 1) * RTCP_PACKET_LEN_WORD_SIZE >= pRawPacketsLen, STATUS_SUCCESS);
```

Comment:
I think this line checks the packet length defined in [RFC 4585 Section 6](https://tools.ietf.org/html/rfc4585#section-6).
Why is this line changed from error to success?
Do you mean C SDK will ignore compound RTCP packet?

Reply:
Yes it does ignore compound packets, we do want to fix this though in the short term.

https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/issues/45 is the ticket.